gnomad_qc/v3/utils.py

```diff
@@ -70,3 +73,28 @@ def rekey_new_reference(
         t = t.key_by("locus", "alleles").drop("locus_original")
 
     return t
+
+
+# TODO: Needs to be modified to fix the problem of the alt het correction on a split MT
+def hom_alt_depletion_fix(
+        mt: hl.MatrixTable,
+        af_expr: hl.expr.Float32Expression,
+        af_cutoff: float = 0.01,
+        ab_cutoff: float = 0.9,
+) -> hl.MatrixTable:
+    """
+    Adjust MT genotypes with temporary fix for the depletion of homozygous alternate genotypes.
+
```

Comment:
might be helpful to add this link to the docstring: https://gnomad.broadinstitute.org/blog/2020-10-gnomad-v3-1-new-content-methods-annotations-and-data-availability/#tweaks-and-updates